packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementListItem.tsx

```diff
@@ -39,15 +42,15 @@ export function AccessRequirementListItem(
         <UnmanagedACTAccessRequirementItem
           accessRequirement={accessRequirement}
           onHide={onHide}
-          entityId={entityId}
+          subjectId={subjectId}
+          subjectType={subjectType}
         />
       )
     case MANAGED_ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE:
       return (
         <ManagedACTAccessRequirementItem
           accessRequirement={accessRequirement}
           onHide={onHide}
-          entityId={entityId}
```

Comment:
Am I missing something?  I thought the subjectId (and type) would be passed into the ManagedACTAccessRequirementItem as well

Reply:
The prop was unused, so I removed it. The relationship between the AR and the subject is handled in the `onRequestAccess` callback.